src/go/k8s/tests/e2e-v2/node-deleted/00-create-redpanda.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: cluster.redpanda.com/v1alpha1
+kind: Redpanda
+metadata:
+  name: redpanda
+spec:
+  chartRef:
+    chartVersion: "5.3.2"
+  clusterSpec:
+    statefulset:
+      replicas: 5
```

Comment:
Can this test perform the same steps, but using replicas 3 not 5?

Reply:
No, if you go lower than 3 it will destabilize the cluster and It will not end in the state we need it to be.